x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list_status.tsx

```diff
@@ -7,10 +7,11 @@
 
 import React from 'react';
 import { EuiIcon } from '@elastic/eui';
-import type { EcsEventOutcome } from '@kbn/core/server';
+
+export type KibanaAlertingOutcome = 'failure' | 'success' | 'unknown' | 'warning';
```

Comment:
nitpick: do you foresee us re-using this type? if so should we move it to a more global location?

Reply:
yea I think it might be better if it lived closer to the event log since its event log outcome types

Reply:
Resolved in b4766c07eb61387606547f99ae9868f1c14c6325